client/app/views/menu_item.coffee

```diff
@@ -64,6 +66,12 @@ module.exports = class MenuItemView extends BaseView
         # Gone after succefull color pick, put it back.
         @$('.dropdown-toggle').on 'click', @hideColorPicker
 
+    onCalendarMultipleSelect: ->
+        actionMenu = $('#multiple-actions')
+        if $('.calendar-actions:checked').length > 0
```

Comment:
I recommend that we open the multi-actions menu only if more than one calendar is selected (we won't perform a bulk action on only one calendar).